pkg/pagerduty/pagerduty.go

```diff
@@ -62,7 +62,38 @@ func (p PagerDuty) MoveToEscalationPolicy(incident Incident, escalationPolicy Es
 	}
 	err := p.manageIncident(o)
 	if errors.Is(err, FailedToUpdateIncidentError{}) {
-		return pkgerrors.Wrap(err, "could not update the escalation policy")
+		return fmt.Errorf("could not update the escalation policy: %w", err)
+	}
+	return err
```

Comment:
wouldn't you want to add "could not update the escalation policy on any error" for the case an error is returned by manageIncident. Or to ask differently, is there a case where an error is returned but the update was nevertheless successful?
```suggestion
	err := p.manageIncident(o)
	if err != nil {
		return fmt.Errorf("could not update the escalation policy: %w", err)
	}
	return nil
```

Reply:
we could maybe think about the following function, just to save some lines:

```suggestion
	return onErrorAddDetails(err, "could not update the escalation policy:")
```

and the onErrorAddDetails would return nil for the case err == nil

Reply:
no need to wrap this, saw your reasoning and raised it.
if we need to handle the `UnknownUpdateIncidentError`, it can be easily done